SimpleTextingDotNet/v2/Endpoints/Media.cs

```diff
@@ -0,0 +1,123 @@
+﻿// <copyright>
+// MIT License
+// 
+// Copyright (c) 2022 Kingdom First Solutions
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+// </copyright>
+
+using SimpleTextingDotNet.v2.Model.Object;
+using SimpleTextingDotNet.v2.Model.Request;
+using SimpleTextingDotNet.v2.Model.Response;
+using RestSharp;
+
+namespace SimpleTextingDotNet.v2
+{
+    public partial class Client
+    {
+        /// <summary>
+        /// Upload a media file from a local file to SimpleTexting.
+        /// </summary>
+        /// <param name="file">The media file you want to upload</param>
+        /// <param name="shared">Define whether a media file is shared with teammates</param>
+        /// <returns>MediaItem</returns>
+        public MediaItem Upload( string file, bool shared = true )
```

Comment:
This should be tested.

Reply:
Tested and it did not work. Fixed in https://github.com/KingdomFirst/SimpleTexting-DotNet-Framework/pull/1/commits/b2d483f7403d8e7ed82f6a9cd80ffb96492dab7a